config/default/grumphp.yml

```diff
@@ -45,11 +45,18 @@ parameters:
 
   git_blacklist.keywords:
     - "die("
+    - "dd("
     - "var_dump("
+    - "console.log("
+    - "alert("
+    - "print_r("
+    - "phpinfo();"
     - "exit;"
-    - "Magento\\\\Framework\\\\App\\\\ObjectManager"
+    - "<<<<<"
+    - "====="
+    - "<?php echo"
```

Comment:
My gut feeling says "<?php echo"  could be all over the place
For those not willing to refactor all their existing templates we might want to describe how to locally over-rule this rule-set. That said, I think we need a page on the git blacklist feature in general to be created (and linked from the README) think I missed this one in my documentation update

Reply:
Added git blacklist docs and added reference in README. Docs contains example on how to override, as well as a reference to the original git blacklist docs for all available configuration keys that can be used :+1: